### DISCUSSION TITLE
Using the cargo sparse registry feature within Crane

### DISCUSSION THREAD

**Discussion by msfjarvis** [ipetkov/crane#261] [Q&A] [answered]
I wish to use the new [sparse registry](https://blog.rust-lang.org/inside-rust/2023/01/30/cargo-sparse-protocol.html) feature for my Crane projects, for which I've made [this change](https://github.com/msfjarvis/adx/commit/7df7ad4d71653905f4e25ad906f8439f1cb87ba1) in my usage of Crane and while this does work within the `devShell`, how can I confirm that the builds being done by Crane itself also use the sparse protocol?

**Comment by ipetkov** [comment, marked as answer]
`crane` does not check out the crates.io index (not directly, anyway, but this only applies if you are using alternative registries)

Crates are vendored directly with the information present in the `Cargo.lock` file
